### ISSUE TITLE
Electronic format

### ISSUE THREAD

**Issue by willettk** [vrooje/gzcandels_datapaper#30]
What's the ultimate release format of the data? The paper says it'll be on http://data.galaxyzoo.org, which we should do, but I want to include it in more formats for posterity's sake. 

Possible options:
- [x] http://data.galaxyzoo.org (CSV/FITS/HDF5/other)
- [ ] supplementary material to astro-ph
- [ ] supplementary material to MNRAS
- [ ] VizieR/CDS
- [ ] MAST
- [ ] CANDELS data access page
- [ ] other?

**Comment by vrooje** [comment]
Also this brings up another issue...

**Comment by vrooje** [comment]
I say data.galaxyzoo.org and VizieR/CDS at a minimum so that people can match to other tables via a cone search.

**Comment by willettk** [comment]
Agreed.

**Comment by vrooje** [comment]
I really want to close this because I want that milestone to be met but then again we won't have the final versions of tables until publication, right?

**Comment by willettk** [comment]
Can split it into sub-issues for each type, if you want

**Comment by vrooje** [comment]
yesssss

**Comment by willettk** [comment]
data.galaxyzoo.org is ready to be deployed (https://github.com/zooniverse/data.galaxyzoo.org/commit/7ae72d66a3abeca0e9fcde9ca77a02b99771af6c) whenever the paper is accepted.

**Comment by vrooje** [comment]
I updated the data.galaxyzoo.org repo (zooniverse/data.galaxyzoo.org@85d21e668fb5a9614807c948497f730d2397fd6e and a few commits before that) to account for the changes made in the resubmission, and also updated the files available on AWS.

I added the commits to the hubble branch as it seemed like that may have been more updated than the candels branch? Hope that was the right thing...

**Comment by willettk** [comment]
Awesome - I'm really sorry I didn't tackle it myself last week.

Yeah - the `hubble` and `candels` branches are diverging somewhat, but I
think they have fairly parallel changes and should be eventually merged
without difficulty. Only the master branch is actually deployed live to the
site, which we said we were waiting on until the papers are accepted or
posted to arXiv. Just give me a shout when you're ready for that.

**Comment by vrooje** [comment]
I'd be happy to make it public now (without advertising it yet), if @chrislintott and @karenlmasters approve...

**Comment by vrooje** [comment]
Update: actually it would be really great to publish this soon as there's a CANDELS meeting next week and they'd like an update on the catalog, so this seems like the easiest way.

**Comment by willettk** [comment]
That's fine by me. I suspect few people will notice, as I don't know how
regularly the site is visited (I don't know that we have Google Analytics
or anything similar implemented for data.galaxyzoo.org, actually).

I'd rather hold off on adding the GZ Hubble data yet, though, as Mel and I
aren't 100% complete yet. I think it would have been more appropriate to
put these changes in the `candels` branch after all.

Do you know if there's a git way that we can peel off your last five
commits and attach them to `candels` instead of `hubble`? Undoing the
`hubble` branch is easy (just a revert), but I want to know if there's a
way we could reapply them to the `candels` branch and then merge that with
`master`.

Maybe the devs can help us out if neither of us know off the top of our
head.

**Comment by karenlmasters** [comment]
Fine by me too.

**Comment by vrooje** [comment]
Okay, thanks to @srallen and `git cherry-pick`, I've moved the commits over to the candels branch and rewound the hubble branch.

**Comment by willettk** [comment]
Awesome sauce. Thanks for the help, @srallen.

Looks like merge into master from this branch will be painless, so feel
free to do so whenever you're ready. Instructions on actually deploying are
in the README.md file; let me know if you have any issues with that
(versions of node or keys can be weird). I can try if it doesn't work for
you, @vrooje.

**Comment by vrooje** [comment]
Great, merged -- but I don't have node and also my key is now only stored in one program in encrypted form and I don't actually know what it is! If you wouldn't mind trying the deploy that would be fantastic... thanks.

**Comment by willettk** [comment]
Tried, but my AWS keys no longer work - I suspect @astopy rightly discontinued them. :(

Looks like we'll have to ask a dev to help (@srallen?) or else get node installed on your machine.

**Comment by vrooje** [comment]
Okay turns out the secret key was also in my keychain and node + nvm aren't that hard to install.

Done.
